common/djangoapps/course_modes/migrations/0005_auto_20151217_0958.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('course_modes', '0004_auto_20151113_1457'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
@chrisndodge The code changes have already been pushed. This shouldn't actually touch the database, since we're only changing field names and not doing any sort of data migration. This file just lets Django keep track of the model-level changes.